grafana-plugin/e2e-tests/users/usersActions.test.ts

```diff
@@ -61,6 +61,8 @@ test.describe('Users screen actions', () => {
     const { page } = adminRolePage;
     await goToOnCallPage(page, 'users');
 
+    await page.waitForTimeout(1000);
```

Comment:
is there a more deterministic way of waiting here? Waiting for a static amount of time feels like it could lead to flaky tests (ex. if for example the tests are run in a Cloud Grafana env and for whatever reasons network latency is a bit higher).

Reply:
From what I tested seems like there's a very fast flicker in the search response right in the beginning, but it is not noticeable no matter how many attempts you give. The timeout is kind of needed cause otherwise it will discard the search results, thus instead of like 1 or 2 results it will show you all, failing the tests. Maybe it could be less than 1000ms.

Worth investigating it at some point but I couldn't figure out what's causing that really